### PR TITLE
Adds the create and export command to podman

### PR DESCRIPTION
**Pull request by limistah**
This shows how to create an image without running a container, and how to export a container.

### CODE REVIEW

File: j.podman.md [hunk 1]
```diff
@@ -136,6 +136,31 @@ ef4b14a72d02ae0577eb0632d084c057777725c279e12ccf5b0c6e4ff5fd598b
 </p>
 </details>
 
+### Creates a container without running/starting it
```
Comment (dgkanatsios) [inline j.podman.md:139]: Can we add the fact that podman is used in the question?

File: j.podman.md [hunk 2]
```diff
@@ -136,6 +136,31 @@ ef4b14a72d02ae0577eb0632d084c057777725c279e12ccf5b0c6e4ff5fd598b
...
+```bash
+:~$ podman create simpleapp # create
+:~$ podman container ls
+```
+
+</p>
+</details>
+
+
+### Exports a container to output.tar file
```
Comment (dgkanatsios) [inline j.podman.md:152]: Can we add the fact that podman is used in the question?

File: j.podman.md [hunk 3]
```diff
@@ -136,6 +136,31 @@ ef4b14a72d02ae0577eb0632d084c057777725c279e12ccf5b0c6e4ff5fd598b
 </p>
 </details>
 
+### Creates a container without running/starting it
+<details><summary>show</summary>
```
Comment (dgkanatsios) [inline j.podman.md:140]: nit: empty line

File: j.podman.md [hunk 4]
```diff
@@ -136,6 +136,31 @@ ef4b14a72d02ae0577eb0632d084c057777725c279e12ccf5b0c6e4ff5fd598b
...
+:~$ podman create simpleapp # create
+:~$ podman container ls
+```
+
+</p>
+</details>
+
+
+### Exports a container to output.tar file
+<details><summary>show</summary>
```
Comment (dgkanatsios) [inline j.podman.md:153]: nit: empty line

**Review by dgkanatsios — CHANGES_REQUESTED** [review]

File: j.podman.md (hunk 1 above)
Comment (limistah) [reply j.podman.md]: I have added the output of the command, would this be sufficient?

File: j.podman.md (hunk 2 above)
Comment (limistah) [reply j.podman.md]: I have added the output of the command, would this be sufficient?

File: j.podman.md (hunk 1 above)
Comment (dgkanatsios) [reply j.podman.md]: Sorry, I meant to add in the question that the answer/script needs to include podman. I don’t see this is clear in the current questions

**Review by dgkanatsios — APPROVED** [review]

**Pull request merged**